lib/emscripten/src/env/mod.rs

```diff
@@ -22,11 +22,23 @@ use wasmer_runtime_core::{
 };
 
 pub fn call_malloc(ctx: &mut Ctx, size: u32) -> u32 {
-    get_emscripten_data(ctx).malloc.call(size).unwrap()
+    get_emscripten_data(ctx)
+        .malloc
+        .as_ref()
+        .unwrap()
+        .call(size)
+        .unwrap()
 }
 
 pub fn call_malloc_with_cast<T: Copy, Ty>(ctx: &mut Ctx, size: u32) -> WasmPtr<T, Ty> {
-    WasmPtr::new(get_emscripten_data(ctx).malloc.call(size).unwrap())
+    WasmPtr::new(
+        get_emscripten_data(ctx)
+            .malloc
+            .as_ref()
+            .unwrap()
+            .call(size)
+            .unwrap(),
+    )
```

Comment:
Since this code is the same as `call_malloc()`, does it makes sense to call it to remove duplicated code?